gradoop-store/gradoop-store-api/src/main/java/org/gradoop/common/storage/predicate/filter/impl/PropLargerThan.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradoop.common.storage.predicate.filter.impl;
+
+import org.gradoop.common.storage.predicate.filter.api.ElementFilter;
+
+/**
+ * property larger than min filter
+ *
+ * @param <FilterImpl> filter implement type
+ */
+public abstract class PropLargerThan<FilterImpl extends ElementFilter>
+  implements ElementFilter<FilterImpl> {
+
+  /**
+   * property key
+   */
+  private final String key;
+
+  /**
+   * property value
+   */
+  private final double min;
```

Comment:
Like I wrote in the other comment, min should be of type `PropertyValue` to allow a comparison between supported types like Date etc.